packages/envied_generator/pubspec.yaml

```diff
@@ -11,9 +11,8 @@ dependencies:
   envied: ^1.0.0
   build: ^2.4.1
   code_builder: ^4.8.0
-  dart_style: ^2.3.2
-  source_gen: ^1.4.0
-  analyzer: ">=5.1.0 <7.0.0"
+  source_gen: ^2.0.0
+  analyzer: ^7.0.0
   recase: ^4.1.0
   equatable: ^2.0.5
 
```

Comment:
You'll need to also import
```yaml
pub_semver: ^2.1.4
```
for https://github.com/petercinibulk/envied/pull/130/files#r1918202517